entrypoints/content.ts

```diff
@@ -0,0 +1,6 @@
+export default defineContentScript({
+  matches: ['*://*.mat.umk.pl/*'],
```

Comment:
Proponuję to ograniczyć do www.mat.umk.pl, bo to nam złapie także moodle, archiwum, itp.